library/queue.po

```diff
@@ -56,22 +56,24 @@ msgstr ""
 "et l'élément ayant la valeur la plus faible est extrait en premier."
 
 #: ../Doc/library/queue.rst:26
-#, fuzzy
 msgid ""
 "Internally, those three types of queues use locks to temporarily block "
 "competing threads; however, they are not designed to handle reentrancy "
 "within a thread."
 msgstr ""
-"En interne, le module utilise des verrous pour bloquer temporairement des "
-"fils d'exécution concurrents. Cependant, il n'a pas été conçu pour être "
-"réentrant au sein d'un fil d'exécution."
+"En interne, ces trois types de files utilisent des verrous pour bloquer "
+"temporairement des fils d'exécution concurrents. Cependant, il n'a pas été "
+"conçu pour être réentrant au sein d'un fil d'exécution."
 
 #: ../Doc/library/queue.rst:30
 msgid ""
 "In addition, the module implements a \"simple\" :abbr:`FIFO (first-in, first-"
 "out)` queue type where specific implementations can provide additional "
 "guarantees in exchange for the smaller functionality."
 msgstr ""
+"De plus, ce module implémente une \"simple\" :abbr:`FIFO (first-in, first-"
+"out)`, dont l'implémentation spécifique fournit plus de garanties au "
+"détriment des fonctionalités."
```

Comment:
je pense que le sens est plutôt ''alors qu'une implémentation spécifique* peut fournir plus de garanties, au détriment des fonctionalités''

* donc pas celle dont il est question ici

Reply:
Je pense qu'il parle de `queue.SimpleQueue` (https://github.com/python/cpython/commit/94e1696d04c65e19ea52e5c8664079c9d9aa0e54), qui semble fournir "plus de garantie" (en fait son lock est réentrant) mais moins de fonctionalités ("Simple queues lack advanced functionality such as task tracking.").

Reply:
Je retourne ma veste :-) 
Ma compréhension est qu'il s'agit de `queue.SimpleQueue` mais les implémentations sont celles que des programmeurs feraient autour de cette queue. Donc, j'écrirais :
"ce module implémente une simple queue FIFO (first-in, first-out), permettant à des implémentations spécifiques d'avoir des garanties supplémentaires au détriment de fonctionnalités moindres"

Reply:
Toujours pas d'accord, dans https://github.com/python/cpython/commit/94e1696d04c65e19ea52e5c8664079c9d9aa0e54 @pitrou a ajouté la classe SimpleQueue et a ajouté la phrase "In addition, the module implements a "simple" :abbr:`FIFO`". La phrase parle donc, pour moi sans ambiguité, de SimpleQueue.

Cette modif survient après https://bugs.python.org/issue14976, qui disait "Queue est buggé" et la réponse d'@pitrou était: implémentons une queue plus robuste (plus de garanties) mais plus simple (moins de fonctionalités).

Si j'ai vrai, la version anglaise devrait clairement lier SimpleQueue dans la phrase, ce que je vais proposer à l'instant.

Reply:
Nous sommes donc à moitié d'accord ! il s'agit bien de `queue.SimpleQueue`.
Les implémentations spécifiques seraient alors celle de Cpython, et les autres (PyPy, ...) ?

Reply:
c.f. https://github.com/python/cpython/pull/8372 en attendant je la laisse "fuzzy".